libs/api/database/src/lib/content/stores/content-group.store.ts

```diff
@@ -205,8 +203,8 @@ export class ContentGroupStore {
             kind: { $in: kinds },
         };
         await ContentGroupStore.determineContentFilter(paginateQuery, filter);
-        return await this.fuzzySearchableContent.fuzzySearch(query);
-        // return await this.content.paginate(paginateQuery, paginateOptions);
+        const fuzzySearchedContent = await this.fuzzySearchableContent.fuzzySearch(query);
```

Comment:
So we're not actually doing anything with this variable. `fuzzySearch()` returns an array of `ContentDocument`s, but this function expects to return a `PaginateResult<ContentDocument>`, and those are two very different things. This is where a big part of the complexity of making `mongoose-paginate` and `mongoose-fuzzy-searching` play nicely together is going to happen.